app/assets/javascripts/discourse/app/lib/sidebar/admin-sidebar.js

```diff
@@ -409,7 +411,27 @@ export default class AdminSidebarPanel extends BaseCustomSidebarPanel {
   }
 
   get searchable() {
-    return true;
+    const currentUser = getOwnerWithFallback(this).lookup(
+      "service:current-user"
+    );
+    return currentUser.admin;
+  }
+
+  get filterable() {
+    const currentUser = getOwnerWithFallback(this).lookup(
+      "service:current-user"
+    );
+    return !currentUser.admin && currentUser.moderator;
+  }
+
+  filterNoResultsDescription(filter) {
+    const escapedFilter = escapeExpression(filter);
+
+    return htmlSafe(
+      i18n("sidebar.no_results.description_admin_search", {
+        filter: escapedFilter,
+      })
+    );
```

Comment:
Do we need `htmlSafe` here if the translation in client.en.yml doesn't contain any HTML?